query.go

```diff
@@ -14,7 +16,7 @@ type QueryService struct {
 	client *Client
 }
 
-func (q *QueryService) Execute(qry builder.Query, result interface{}) (*Response, error) {
+func (q *QueryService) Execute(qry builder.Query, result interface{}, headers http.Header) (*Response, error) {
```

Comment:
```suggestion
func (q *QueryService) Execute(qry builder.Query, result interface{}, headers ...http.Header) (*Response, error) {
```

can we make it optional?

Reply:
sounds good. WDYT @jmichalak-fluxninja ?